services/hockeyapp/hockeyapp.tester.js

```diff
@@ -25,7 +25,7 @@ t.create('Android App')
         ],
       })
   )
-  .expectJSONTypes({ name: 'hockeyapp', value: isVPlusDottedVersionAtLeastOne })
+  .expectJSON({ name: 'hockeyapp', value: isVPlusDottedVersionAtLeastOne })
```

Comment:
Just to clarify my previous comment:

We use `.expectJSON()` when comparing to an object literal e.g: `.expectJSON({ name: 'hockeyapp', value: 'invalid' })` but `.expectJSONTypes()` is necessary when we're validating against an expression, so these ones still need to be `.expectJSONTypes()`. You'll probably find these are failing.

Sorry this is a bit fiddly

Reply:
Thanks for clarifying. Will fix this.